release notes/v0.37.0.md

```diff
@@ -0,0 +1,96 @@
+
+:information_source: If you are an extension developer, please use it and give your feedback. But take into consideration that it's likely that the current Go API may change.
+
+### Added an option to output k6 logs to a file through --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+This is on top of the already supported options for sending logs to stdout/stderr and to [Grafana Loki](https://grafana.com/oss/loki/). This new feature speaks for itself with simple usage examples:
+
+```sh
+k6 run --log-output file=./k6.log --logformat json ./examples/stages.js
+```
+
+And one more with a defined minimal log level:
+
+```sh
+k6 run --log-output file=./k6.log,level=info --logformat json ./examples/stages.js
+```
+
+Thanks, @[alyakimenko](https://github.com/alyakimenko) for the contribution!
+
+**Docs**: [Using file output](https://k6.io/docs/using-k6/options/#file)
+
+### Extensions
+
+`v0.37.0` finalizes ([#2376](https://github.com/grafana/k6/pull/2376)) the switching of our internal modules to [a new Go/JavaScript module API](https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension).  Lastly, the internal gRPC module was refactored ([#2365](https://github.com/grafana/k6/pull/2365)).
+
+:warning: It's important to highlight that the old API (e.g. methods like `context.WithRuntime`, `common.Bind` and others [#2384](https://github.com/grafana/k6/pull/2384)) is deprecated and will be removed in the next k6 release (`v0.38.0`). For this release, every extension that isn't using the new API will get a warning message like this:
+
+```sh
+WARN[0000] Module 'k6/x/sql' is using deprecated APIs that will be removed in k6 v0.38.0, for more details on how to update it see https://k6.io/docs/extensions/guides/create-an-extension/#advanced-javascript-extension
+```
+
+We did migrations for some xk6 extensions (see connected issues to the task [#2344](https://github.com/grafana/k6/issues/2344)). The pull requests can serve as examples on how to transition your extension to the new API.
+
+### Docker Repository
+
+We migrated our Docker Hub repository from `loadimpact/k6` to [grafana/k6](https://hub.docker.com/r/grafana/k6) ([#2377](https://github.com/grafana/k6/pull/2377)).
+
+```sh
+docker run -i grafana/k6 run - <script.js
+```
+
+We will continue publishing our docker image releases as both `loadimpact/k6` and `grafana/k6` for several more releases, but if you use the old one in your local or CI environments, please plan the migration.
+
+## Breaking changes
+
+### Introduced stricter thresholds parsing ([#2400](https://github.com/grafana/k6/pull/2400))
+
+With this change k6 started to return an error message on thresholds that wouldn't have been evaluated correctly, instead of just silently ignoring them.
```

Comment:
```suggestion
In the past, thresholds were evaluated using a Javascript runtime. For a multitude of reasons, this wasn't satisfying. As of **v0.38.0**, thresholds are now parsed directly in Go. As a result, k6 will now return an error message on thresholds that do not strictly match [the documented specification](https://k6.io/docs/using-k6/thresholds/), instead of just silently ignoring them. Another change is that when a non syntaxically correct threshold expression is detected, k6 will immediately interupt its execution before even starting the load test run.
```

Reply:
we are releasing `0.37.0` not `0.38.0` ;)

Reply:
For the record, I've updated @oleiade 's comment in order to include it to a changes batch